rust/trezor-tjpgdec/README.md

```diff
@@ -0,0 +1,10 @@
+# trezor-tjpgdec
+
+Rust version of [Tiny JPEG Decompressor](http://elm-chan.org/fsw/tjpgd/00index.html).
+
+## Credits
+
+* Original C library was written by [ChaN](http://elm-chan.org/).
+* Several bugs fixed by [Chris Mumford](https://github.com/cmumford/TJpgDec).
+* Converted to Rust using [c2rust](https://c2rust.com/) by [Martin Milata](mailto:martin.milata@satoshilabs.com).
+* Heavy refactoring by [Michal Bruna](mailto:michal.brun@satoshilabs.com).
```

Comment:
Missing `a` from Michal's surname in email :)

Reply:
omg, fixed